package/doc/sphinx/source/documentation_pages/converters.rst

```diff
@@ -26,12 +26,25 @@ you will have to specify a package name (case-insensitive). ::
 
     pgro2 = ugro.atoms.convert_to('PARMED')  # converts back to parmed structure
 
+Another syntax is also available for tab-completion support::
+
+    pgro2 = ugro.atoms.convert_to.parmed()
+
 
 .. rubric:: Available converters
 
 .. toctree::
    :maxdepth: 1
 
-   converters/ParmEdParser
-   converters/RDKitParser
+   converters/init
```

Comment:
sphinx was complaining about the presence of `init.rst` without a toctree entry for it. I added it, although it isn't super informative as an entry, should we just remove it?

Reply:
Alternatively, we could move the converters.rst contents into converters.__init__ ?

Reply:
Went with the path of least resistance here and just moved a tiny bit of __init__'s doc to the `converters.rst`. We can always clean up in 2.1.0 (or post beta) if it's really worth it.